ignite/handlers/checkpoint.py

```diff
@@ -57,8 +57,8 @@ class Checkpoint:
     storage, etc.
 
     Args:
-        to_save (Mapping): Dictionary with the objects to save. Objects should have implemented `state_dict` and `
-            load_state_dict` methods.
+        to_save (Mapping, optional): Dictionary with the objects to save. Objects should have implemented `state_dict`
```

Comment:
It is not optional and should not be for public API, but for refactoring sake with `ModelCheckpoint`, it can accept None.

Reply:
Ok i thought it was a missing optional.